content/en/docs/next/demo.md

```diff
@@ -245,18 +245,16 @@ etcdctl --endpoints=$ENDPOINTS elect one p2
 
 Specify the initial cluster configuration for each machine:
 
-![10_etcdctl_endpoint_2016050501](https://storage.googleapis.com/etcd/demo/10_etcdctl_endpoint_2016050501.gif)
-
 ```shell
 etcdctl --write-out=table --endpoints=$ENDPOINTS endpoint status
 
-+------------------+------------------+---------+---------+-----------+-----------+------------+
-|     ENDPOINT     |        ID        | VERSION | DB SIZE | IS LEADER | RAFT TERM | RAFT INDEX |
-+------------------+------------------+---------+---------+-----------+-----------+------------+
-| 10.240.0.17:2379 | 4917a7ab173fabe7 | 3.0.0   | 45 kB   | true      |         4 |      16726 |
-| 10.240.0.18:2379 | 59796ba9cd1bcd72 | 3.0.0   | 45 kB   | false     |         4 |      16726 |
-| 10.240.0.19:2379 | 94df724b66343e6c | 3.0.0   | 45 kB   | false     |         4 |      16726 |
-+------------------+------------------+---------+---------+-----------+-----------+------------+
++------------------+------------------+---------+---------+-----------+------------+-----------+------------+--------------------+--------+
+|    ENDPOINT      |        ID        | VERSION | DB SIZE | IS LEADER | IS LEARNER | RAFT TERM | RAFT INDEX | RAFT APPLIED INDEX | ERRORS |
++------------------+------------------+---------+---------+-----------+------------+-----------+------------+--------------------+--------+
+| 10.240.0.17:2379 | 4917a7ab173fabe7 |  3.4.15 |   45 kB |      true |      false |         4 |      16726 |              16726 |        |
```

Comment:
The `next` folder is for the upcoming v3.5 release. If we use the v3.5.0-alpha.0 release, is this info the same?

Reply:
`HEAD is now at 60d515909 version: bump up to 3.5.0-alpha.0
`


![image](https://user-images.githubusercontent.com/22524871/118343169-e2663d00-b559-11eb-83b6-b020ed26e0f1.png)
Hi, this code is tag `3.5.0-alpha.0`. The column already be more than before.

Reply:
And since `tag` 3.4.0, the docs is not match code.

Reply:
`HEAD is now at 898bd1351 version: 3.4.0`.
![image](https://user-images.githubusercontent.com/22524871/118343311-d75fdc80-b55a-11eb-99ce-8c859b59ab49.png)
3.4.0 still mismatch.

Reply:
It looks like v3.5.0-beta.3 has corrected this mismatch

```
$ETCD_BIN/etcdctl endpoint status
127.0.0.1:2379, 8e9e05c52164694d, 3.5.0-beta.3, 20 kB, true, false, 3, 8, 8,
```

Reply:
Since this is for an upcoming release, maybe we can just put `3.5.0` in, as by the time the release is out this will be correct.

Reply:
> It looks like v3.5.0-beta.3 has corrected this mismatch
> 
> ```
> $ETCD_BIN/etcdctl endpoint status
> 127.0.0.1:2379, 8e9e05c52164694d, 3.5.0-beta.3, 20 kB, true, false, 3, 8, 8,
> ```

The code since 3.4.0 already append some column, but the docs v3.4, v3.5-DRAFT is not match. It's not code problem, it's docs problem.